src/app/bookings/bookings.component.html

```diff
@@ -27,7 +36,7 @@
 
     <ng-container matColumnDef="arrivalTime">
       <th mat-header-cell *matHeaderCellDef> Arrival Time </th>
-      <td mat-cell *matCellDef="let element"> {{element.flight.arrivalTime}} </td>
+      <td mat-cell *matCellDef="let element"> {{dateFormated(element.flight.arrivalTime)}} </td>
```

Comment:
We are implementing this in different ways, lets get together to figure out which one to use

Reply:
agreed. After review lets use your formating for the dateTime